src/llmq/quorums_instantsend.cpp

```diff
@@ -934,13 +934,11 @@ void CInstantSendManager::UpdateWalletTransaction(const CTransactionRef& tx, con
         return;
     }
 
-    if (pwalletMain->UpdatedTransaction(tx->GetHash())) {
```

Comment:
same

Reply:
can you clarify what you mean?

Reply:
https://github.com/dashpay/dash/pull/2974#discussion_r293877228 fixes the same issue